pkg/tenant/members.go

```diff
@@ -67,21 +98,29 @@ func (s *Server) TenantMemberCreate(c *gin.Context) {
 	c.JSON(http.StatusCreated, member)
 }
 
+// MemberList retrieves all members assigned to a tenant
+// and returns a 200 OK response.
+//
+// Route: /member
 func (s *Server) MemberList(c *gin.Context) {
-	// The following TODO task items will need to be
-	// implemented for each endpoint.
+	// TODO: Fetch the member's tenant ID from key.
+
+	// Get members from the database and return a 500 response
+	// if not succesful.
+	if _, err := db.ListMembers(c.Request.Context(), ulid.ULID{}); err != nil {
+		log.Error().Err(err).Msg("could not fetch members from database")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not fetch members from database"))
+		return
+	}
+
+	// Build the response.
+	out := &api.MemberPage{Members: make([]*api.Member, 0)}
 
-	// TODO: Add authentication and authorization middleware
-	// TODO: Identify top-level info
-	// TODO: Parse and validate user input
-	// TODO: Perform work on the request, e.g. database interactions,
-	// sending notifications, accessing other services, etc.
+	member := &api.Member{}
 
-	// Return response with the correct status code
+	out.Members = append(out.Members, member)
```

Comment:
I have the same question here as in the `TenantMemberList` function -- it seems like we should be using the slice of `members` the database passes back to us on line 110 instead of assigning it to `_`, and then perhaps assigning `out.Members = members` rather than appending an empty `member`